sympy/ntheory/primetest.py

```diff
@@ -110,6 +110,19 @@ def mr(n, bases):
     return True
 
 
+def isprimeE(n,b):
```

Comment:
Here and below, there should be a space after each comma

Reply:
This is readable and follows basic standards of code